doc/radosgw/config-ref.rst

```diff
@@ -416,6 +382,69 @@ instances or all radosgw-admin commands can be put into the ``[global]`` or the
 :Default: ``true``
 
 
+Garbage Collection Settings
+===========================
+
+The Ceph Object Gateway allocates storage for new and overwritten objects
+immediately.
+
+The Ceph Object Gateway purges the storage space used for deleted objects in the
```

Comment:
i think the part about overwrites is potentially confusing here. we allocate storage for new objects immediately, and garbage collect objects that were deleted and ovewritten

so i'd suggest moving the `and overwritten` part from the first sentence to the second one, ie `new objects immediately` and `for deleted and overwritten objects`